lib/quadquizaminos_web/live/tetris_live.ex

```diff
@@ -37,7 +35,7 @@ defmodule QuadquizaminosWeb.TetrisLive do
      |> start_game()}
   end
 
-  def render(%{state: :starting, live_action: live_action} = assigns) do
+  def render(%{state: :starting, live_action: _live_action} = assigns) do
```

Comment:
```suggestion
  def render(%{state: :starting} = assigns) do
```
No need of pattern matching with `live_action` since its no longer been used

Reply:
Okay